src/lib/y2storage/dialogs/guided_setup/select_volumes_disks.rb

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE.
+#
+# To contact SUSE about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2storage"
+require "y2storage/dialogs/guided_setup/base"
+require "y2storage/dialogs/guided_setup/widgets/disk_selector"
+
+module Y2Storage
+  module Dialogs
+    class GuidedSetup
+      # Dialog for volumes disks selection for the proposal.
+      class SelectVolumesDisks < Base
+        def initialize(*params)
+          textdomain "storage"
+          super
+        end
+
+        # This dialog has to be skipped when there is only
+        # one candidate disk for installing.
+        def skip?
+          analyzer.candidate_disks.size == 1
+        end
+
+        # Before skipping, settings should be assigned.
+        def before_skip
+          settings.candidate_devices = analyzer.candidate_disks.map(&:name)
+        end
+
+        protected
+
+        def dialog_title
+          _("Select Hard Disk(s)")
+        end
+
+        def dialog_content
+          content = disk_selector_widgets.flat_map { |w| [w.content, VSpacing(1.4)] }.tap(&:pop)
```

Comment:
As we agreed in a short talk, I am simply going to put a TODO there, since it seems that it deserves a kind of UI helper but, at the same time, it is a bit out of the scope of this PR.